async-inference/HuggingFace-Async-Inference-Walkthrough.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>Add brackets to show where terms should be replaced, such as <code>"arn:aws:s3:::&lt;bucket_name&gt;/*"&nbsp;</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2910/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Have updated the brackets accordingly.